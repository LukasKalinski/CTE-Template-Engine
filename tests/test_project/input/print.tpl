{$otherFruit->friends.peter}

{$foo[$magicKey].message}

{$foo[$magicKey]}

{$fruit->color}

{$fruit->slice(4)}

{$foo.bar}

{$foo.bar.message}

{2 * (2 + (10 * 2)) * 2 - 11}

{$cte.time + (77 * 10)}

{* Variables with modifiers *}
{$user|upper}

{$user|upper|default:'N/A'}

{* Resources ... *}
