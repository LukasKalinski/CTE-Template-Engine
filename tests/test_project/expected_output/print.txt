<?php include_once('CTE/Plugin/Default.php');?>
<?php $this->v_plugin['default']=new CTE_Plugin_Default();?>


<?php echo $this->v_initial['otherFruit']->friends['peter'];?>

<?php echo $this->v_initial['foo'][$this->v_initial['magicKey']]['message'];?>

<?php echo $this->v_initial['foo'][$this->v_initial['magicKey']];?>

<?php echo $this->v_initial['fruit']->color;?>

<?php echo $this->v_initial['fruit']->slice(4);?>

<?php echo $this->v_initial['foo']['bar'];?>

<?php echo $this->v_initial['foo']['bar']['message'];?>

77

<?php echo time()+(77*10);?>

<?php echo strtoupper($this->v_initial['user']);?>

<?php echo $this->v_plugin['default']->apply(strtoupper($this->v_initial['user']),array('N/A'));?>